### DISCUSSION TITLE
Splat route

### DISCUSSION THREAD

**Discussion by Fristi** [dream11/zio-http#294] [Q&A]
Hi there!

I would like to have a splat route such that I can request like:

```
GET /static/js/script.js
```

With the following:

```scala
case _ -> Root / "static" / path        => fromStatic(path)
```

In this case `path` would be `js/script.js`, but it seems the path is being splitted with the slashes

**Comment by tusharmath** [comment]
This is supported. You can use `/` `/:`

**Reply by Fristi** [reply]
Thanks for your answer!

In which version is it ? I'm using 1.0.0-RC17

I've changed it to 

```scala
case _  -> Root / "static" /: path => ???
```

But it fails to compile

```
left- and right-associative operators with same precedence may not be mixed
```

**Reply by tusharmath** [reply]
Try RC 19.
